### PR TITLE
refactor!: changes to help with serverless (#82)

### LINKED ISSUE #82: Transition to serverless
We'll be moving away from Elastic Beanstalk. This requires a few things to help clean up environments + improve performance:

- [x] Remove Elastic Beanstalk code / files 
- [x] Refactor app environment variables
- [x] Separate out dev dependencies

### PR DESCRIPTION
**Pull request by korikuzma**
Close #82

@jsstevenson I'm currently loading the ddb tables. If things crash, I will make additional changes. But think the PR is in a good shape to review. 

Notes:
- Remove Elastic Beanstalk code / files
- Refactor app environment variables
- Separate out dev dependencies + pin ga4gh.vrsatile.pydantic

### CODE REVIEW

**Comment by korikuzma** [comment]
@jsstevenson no errors on loading the ddb 👍

File: disease/cli.py [hunk 1]
```diff
@@ -1,14 +1,27 @@
...
+from disease import SOURCES_LOWER_LOOKUP, logger
+from disease.schemas import SourceName
+from disease.database import Database, confirm_aws_db_use, SKIP_AWS_DB_ENV_NAME, \
+    VALID_AWS_ENV_NAMES, AWS_ENV_VAR_NAME
+from disease.etl.merge import Merge
+from disease.etl.mondo import Mondo
+from disease.etl import NCIt  # noqa: F401
+from disease.etl import DO  # noqa: F401
+from disease.etl import OncoTree  # noqa: F401
+from disease.etl import OMIM  # noqa: F401
```
Comment (jsstevenson) [inline disease/cli.py:18]: Can this be a single import statement (and is something goofed about Mondo that won't let it import from the toplevel `disease.etl` namespace)?

**Review by jsstevenson — APPROVED** [review]

File: disease/cli.py (hunk 1 above)
Comment (korikuzma) [reply disease/cli.py:18]: Oh ya, sorry I just copied/pasted from what was in disease/__init__.py

Comment (jsstevenson) [reply disease/cli.py:18]: Yeah, that's what I figured. I can tweak it in a future PR, no biggie

Comment (korikuzma) [reply disease/cli.py:18]: Wow, not all heroes wear capes! Thanks.

**Pull request merged**